README.md

```diff
@@ -1,44 +1,39 @@
 
 ---
 
-<table style="border: 0px;">
-  <tr>
-    <td align="left"><strong>EDGI participated in <a href="https://summerofcode.withgoogle.com/organizations/6574993514168320/">Google Summer of Code</a> for the first time in 2017</strong>! <br /> Our mentoring team of six volunteer EDGI members worked with two students who contributed to our Archiving and Website Monitoring projects, you can read more <br /> about their experience on our website: <a href="https://envirodatagov.org/summer-of-code/">envirodatagov.org/summer-of-code/</a></td>
-    <td align="right"><img src="https://developers.google.com/open-source/gsoc/resources/downloads/GSoC-icon-192.png" /></td>
-  </tr>
-</table>
-
 ## Table of Contents
 
 - [Get Involved](#get-involved)
-- [~Weekly Community Calls~](#weekly-community-calls)
 - [Projects](#projects)
 - [Roadmap](#roadmap)
 - [Working Openly](#working-openly)
 
 ## Get Involved
 
-If you'd like to join the community and help improve these tools, great!
+Welcome to our community! We welcome contributors from many skillsets. Here's how to get started:
 
-1. Review our [Contributor Guidelines](/CONTRIBUTING.md) and [Code of Conduct](/CONDUCT.md)
-1. Jump on the [Archivers chat (archivers.slack.com)](https://archivers.slack.com/), anyone can request an invite from [archivers-slack.herokuapp.com slackin](https://archivers-slack.herokuapp.com/)
-    - Introduce yourself in `#introductions`, key starting places for conversations are `#general`, `#datatogether` and `#community-building`
+1. **Review our [Contributor Guidelines](/CONTRIBUTING.md) and [Code of Conduct](/CONDUCT.md)**
+1. **Jump on the [Archivers chat (archivers.slack.com)](https://archivers.slack.com/)**. Sign up for an account [here](https://archivers-slack.herokuapp.com/) (open to all).
+    - **Introduce yourself** in `#introductions`, key starting places for conversations are `#general`, `#datatogether` and `#community-building`
     - Ping one of the EDGI coordinators (@lightandluck or @kelsey) with your GitHub name to be added to the organization
-1. You can also ping our coordinators on Github at @lightandluck or @Frijol. Please let us know a bit about yourself and what you're interested in.
-1. Take a look at our [Current Projects](#projects), [Roadmap](#roadmap) and [Kanban Board](https://github.com/edgi-govdata-archiving/overview/projects/2)
-
-**Note for IRC users:** (Advanced) If you prefer to use an IRC client, please review these [configuration instructions for Slack's IRC gateway](https://archivers.slack.com/account/gateways).
+1. You can also **ping our coordinators** on Github at @lightandluck or @Frijol. Please let us know a bit about yourself and what you're interested in.
+1. **Take a look at our [Current Projects](#projects)**, [Roadmap](#roadmap) and [Kanban Board](https://github.com/edgi-govdata-archiving/overview/projects/2)
```

Comment:
Remove `Roadmap` link because we removed the section below.

Not sure if you plan to resurrect the Kanban board, so not sure if that link should also be removed.

Reply:
Oh good catch! Will do.